srtcore/api.cpp

```diff
@@ -2805,9 +2759,9 @@ int cleanup()
    return CUDT::cleanup();
 }
 
-SRTSOCKET socket(int af, int type, int protocol)
+SRTSOCKET socket(int , int , int )
 {
-   return CUDT::socket(af, type, protocol);
+   return CUDT::socket();
 }
```

Comment:
Can we just remove the arguments? They are not used anymore.
This changes the UDT API a bit, but still do we need to stick to it?

Reply:
No. There's completely no reason for changes in the legacy UDT API.

Although I can reform it so that this function exactly isn't defined directly.